tests/s3/test_bucket_tagging.py

```diff
@@ -757,6 +757,17 @@ def test_2451(self):
             self.log.info(error)
             assert_utils.assert_in(errmsg.NO_BUCKET_OBJ_ERR_KEY, str(error.message), error.message)
         self.log.info("Step 1: Setting a tag for non existing bucket failed with: NoSuchBucket")
+        self.log.info("Step 2: Retrieving tag of non existing bucket")
+        try:
+            resp = self.tag_obj.get_bucket_tags(self.bucket_name)
+            self.log.info(resp)
+            assert_utils.assert_false(resp[0], resp[1])
+        except CTException as error:
+            self.log.info(error)
```

Comment:
```suggestion
            self.log.exception(error)
```

Reply:
Updated changes and validated against new code changes